_usage/command-line.md

```diff
@@ -183,7 +183,7 @@ x
 ```
 
 There are a couple of things worth mentioning here: First of all, we have to use
-<code>GO</code> to make Babelfish execute the command. This is just like in standard MS SQL,
+<code>GO</code> to make Babelfish execute the command. This is just like in standard Microsoft SQL Server,
 as well as in Sybase. But there is more, which might be alien to users
```

Comment:
I don't think we need the word 'standard', and we shouldn't be referencing Sybase (since this project has nothing to do with Sybase):

As when using Microsoft SQL Server, don't forget to use the <code>GO</code> command after entering one or more SQL statements to instruct Babelfish to execute your commands.